### PR TITLE
migrate user manager documentation

### PR DESCRIPTION
**Pull request by kunkun-tang**

### CODE REVIEW

File: docs/userManager.rst [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+UserManager
+==================================
+
+When you start Azkaban, you may notice the login page. Azkaban makes you authenticate before you can use it.
+This is prevent seeing or executing workflows you shoudn’t see or touch.
+
+We also used authenticated users for auditing purposes. Whenever project files change, is modified, scheduled, etc.
+we often want to know which user performed that action.
+
```
Comment (weidious) [inline docs/userManager.rst]: A figure is missing.

File: docs/index.rst [hunk 2]
```diff
@@ -35,6 +35,7 @@ Features
    configuration
    userGuide
    useAzkaban
+   userManager
    eventTrigger
```
Comment (weidious) [inline docs/index.rst]: Shouldn't this userManager exist above useAzkaban?

Comment (kunkun-tang) [reply docs/index.rst]: Good catch! WIll fix

File: docs/userManager.rst (hunk 1 above)
Comment (kunkun-tang) [reply docs/userManager.rst]: Figure is just added.

**Pull request merged**